int8 averagePosition
byte rawData
int8 Density
